decidim-home/app/views/layouts/_footer.html.erb

```diff
@@ -1,18 +1,35 @@
 <div class="mini-footer">
-	<div class="row">
-		<div class="medium-2 small-12 column">
-			<a rel="license" class="cc-badge part-logo-footer-gencat"
-				 href="http://www.gencat.cat" target="_blank">
-					<%= image_tag("logo_generalitat_gris.png")%>
-			</a>
-		</div>
-		<div class="medium-10 small-12 column">
-			<div class="part-justify part-avis-legal">
-				<a href="http://web.gencat.cat/<%= I18n.locale %>/menu-ajuda/ajuda/avis_legal/" target="_blank">
-						<%= t("static.footer.legal_advice_title")%>:
-				</a>
-				<%= t("static.footer.legal_advice_text")%>
-			</div>
-		</div>
-	</div>
+  <div class="row">
```

Comment:
Can you rename `layouts/_footer.html.erb` to `layouts/_main_footer.html.erb' and add the new content into a new partial `layouts/_mini_footer.html.erb` invoked below?

Reply:
Done https://github.com/gencat/participa/pull/108/commits/7caa7c90b1b7160853146fd90ff41583b67f20e4